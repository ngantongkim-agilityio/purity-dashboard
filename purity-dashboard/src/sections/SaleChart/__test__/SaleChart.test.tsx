import { render } from '@testing-library/react';

// Components
import { SaleChart } from '..';

jest.mock('@/services', () => ({
  fetchRevenue: jest.fn(),
  fetchLatestProducts: jest.fn()
}));

describe('SaleChart component', () => {
  it('Should render SaleChart component with default props', () => {
    const { container } = render(<SaleChart />);

    expect(container).toMatchSnapshot();
  });
});
