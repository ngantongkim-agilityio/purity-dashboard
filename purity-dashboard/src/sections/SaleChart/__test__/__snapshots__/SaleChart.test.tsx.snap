// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SaleChart component Should render SaleChart component with default props 1`] = `<div />`;
